### ISSUE TITLE
active communiting and climate

### ISSUE THREAD

**Issue by mpadge** [gmost/gmost-project#2]
We've said we'd have to leave this issue, but [this tweet](https://mobile.twitter.com/bikesRdata/status/913767915199090688) got me thinkin ... If we could gather enough bike counting stations dumping hourly totals, they could be related to local weather to get a patchy but hopefully decent global-ish relationship between weather and active commuting. We'd need to do the bloody R package for bike counting stations then.

Thanks @maelle
ping @Robinlovelace @MilesMcBain

**Comment by Robinlovelace** [comment]
Yes but I think peoples' reactions to climate is culturally conditioned, i.e. not objective and potentially malleable, so skeptical of the benefit of basing future potential on that measure. People considering cycling in Copenhagen react differently to rain/snow than people in Spain, for example, partly due to clothing, which is changeable.

**Comment by mpadge** [comment]
all cool then. i didn't wanna do it anyway (bike count package, that is), just needed some kind of compelling reason not to.  Let's just continue on as climate deniers. :smirk: ... unless @Robinlovelace can think of any other potential use of such data? Global snapshot of possible ranges of cycling activity to total population? Shit, that's probably really useful anyway in its own right, right? Infrastructure via OSM; crude cycling measures through stationary counters?

**Comment by Robinlovelace** [comment]
Indeed, the more evidence we have on the relative power of different explanatory variables the better in different places. Then when performing the dark art of estimating future potential, as www.pct.bike and TfL do, we can select which of those variables (preferably the most transparent and unchanging) we should include.

**Comment by maelle** [comment]
I agree regarding culture but then maybe one could see an effect of worse weather compared to usual weather as opposed to absolute bad weather?

**Comment by mpadge** [comment]
yeah @maelle, that's kinda what i was thinking, but have in my head the differences between sub-equatorial African (in both directions), SE Asia, and the middle east. Not many large-scale environmental variables (lat, lon, elevation, ...) would at all account for differences in cycling behaviour between those 3/4 places. Probably better to start with a uniform estimate of variability in cycling behaviour before we even try to introduce hypothetical explanatory variables. Whatever the result, looks like your tweet-ping might have had the expected effect of prompting another bikesRdata package ...

**Comment by maelle** [comment]
What a productive tweet ping 😀

**Comment by mpadge** [comment]
yeah, right? @hrbrmstr extolls [SODD](https://rud.is/b/2017/09/28/sodd-stackoverflow-driven-development/) (SO Driven Dev). This is one step down: TWEEDD
